src/System.Net.Security/ref/System.Net.Security.csproj

```diff
@@ -2,10 +2,29 @@
   <PropertyGroup>
     <ProjectGuid>{A7488FC0-9A8F-4EF9-BC3E-C5EBA47E13F8}</ProjectGuid>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
+    <!-- This is needed so that code for TlsCipherSuite will have no namespace (causes compile errors) when used within T4 template  -->
```

Comment:
I'm not understanding this.  Can you elaborate?  Why is this an issue with this code but not with other code we use T4 with, like Vector.tt?

Reply:
in here there are 2 T4 files:
- TlsCipherSuite
- TlsCipherSuiteData

in here TlsCipherSuiteData depends on the generated TlsCipherSuite in other places they either depend on each other or have no dependencies (T4 on T4 - not T4 on generated .cs)